{"name": "Создание заказа при неверном хеше ингредиентов", "status": "passed", "start": 1701266581688, "stop": 1701266581935, "uuid": "c7aabed9-fb73-4eab-b16b-bd858d16b122", "historyId": "1ef5c72fd23a8885ac873ba0deb08da9", "testCaseId": "1ef5c72fd23a8885ac873ba0deb08da9", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_wrong_hash_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "i101922157"}, {"name": "thread", "value": "16636-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}